alignak/objects/contactgroup.py

```diff
@@ -73,6 +73,8 @@ class Contactgroup(Itemgroup):
         'uuid':                 StringProp(default='', fill_brok=['full_status']),
         'contactgroup_name':    StringProp(fill_brok=['full_status']),
         'alias':                StringProp(fill_brok=['full_status']),
+        'contactgroup_members': ListProp(default=[], fill_brok=['full_status'],
```

Comment:
There is a syntax error on the line you added ... missing parenthesis to close the ListProp!